src/varname.jl

```diff
@@ -93,11 +93,13 @@ function Base.show(io::IO, vn::VarName)
     print(io, getsym(vn))
     for indices in getindexing(vn)
         print(io, "[")
-        join(io, indices, ",")
+        join(io, map(replace_colon_string, indices), ",")
         print(io, "]")
     end
 end
 
+replace_colon_string(x) = x
+replace_colon_string(::Colon) = ":"
```

Comment:
Would it make sense to call this something like `prettyprint_index`, to allow for later extensibility?  E.g., I have thought about using negative indexing in my other PR, and then you'd want to have something like 
```julia
prettyprint_index(ix::InvertedIndex) = "-$(prettyprint_index(ix.skip))"
```

Reply:
Yeah, I'm happy with that:)